src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/CSharpConvertBinder.cs

```diff
@@ -121,7 +122,13 @@ public override DynamicMetaObject FallbackConvert(DynamicMetaObject target, Dyna
             {
                 return com;
             }
+#else
+            if (!BinderHelper.IsWindowsRuntimeObject(target) && target.LimitType.IsCOMObject)
+            {
+                throw ErrorHandling.Error(ErrorCode.ERR_DynamicBindingComUnsupported);
+            }
```

Comment:
Please consider extracting a helper method (in `BinderHelper`?) to avoid repeating this in each binder method.